01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+# -*- coding: utf-8 -*-
+
+number = (1..20).to_a
+
+number.each do |num|
```

Comment:
```suggestion
(1..20).each do |num|
```

[nits] number の変数は使い回していない様子、また、`to_a` しなくても動くので、こうまとめてしまうことも可能です〜 👍

Reply:
@betachelsea ありがとうございます！なるほど、勉強になりました。変数を使う場合は、相応の理由と言いますか、用途がないと、かえって無駄や混乱を招いてしまうかもしれないですね。

この後に修正をプッシュしつつ、次へ進んでいきたいと思います。